docs/clustertree/README.md

```diff
@@ -0,0 +1,76 @@
+# ClusterTree(集群容器编排)
```

Comment:
Is it okay to put this doc here? what do you think @duanmengkk

Reply:
We need to redesign the entire document.